propolis/src/hw/pci/device.rs

```diff
@@ -974,8 +974,10 @@ impl MsixCfg {
             entries.push(migrate::MsixEntryV1 {
                 addr: lentry.addr,
                 data: lentry.data,
-                is_pending: lentry.pending,
                 is_vec_masked: lentry.mask_vec,
+                is_func_masked: lentry.mask_func,
+                is_enabled: lentry.enabled,
```

Comment:
Shouldn't we re-hydrate the masked-by-function and enabled state from `MsixStateV1`?

Reply:
Ah, so the per-entry values should always reflect the overall value in MsixCfgState/MsixStateV1?

Reply:
Yeah.  That state has been propagated down to the entries so that they needn't consult the device-wide MSI-X config when making a decision about if an interrupt is masked or not.